test/integration/integration_test.go

```diff
@@ -380,6 +385,23 @@ func shouldBeInValidFormat(commandField, format string) error {
 		if net.ParseIP(result) == nil {
 			return fmt.Errorf("%s of previous command is not a valid IP address: %s", commandField, result)
 		}
+	case "IP with port number":
+		split := strings.Split(result, ":")
```

Comment:
I split the content of each case statement into its own function. Small functions is the aim. Makes understanding and testing the code easier. 

A propos testing, we maybe should start writing unit test for these functions ;-)